src/CommonGLPI.php

```diff
@@ -1050,7 +1050,7 @@ class='btn btn-sm btn-icon btn-ghost-secondary'
             if ($this instanceof CommonITILObject) {
                 echo "<h3 class='navigationheader-title strong d-flex align-items-center'>";
                 echo "<i class='" . $this->getIcon() . " me-1'></i>";
-                echo $this->getName();
+                echo $this->getId() . ' - ' . $this->getName();
```

Comment:
Since a parameter exists to hide/display IDs, I guess it should be used:
```suggestion
                echo ($_SESSION["glpiis_ids_visible"] ? $this->getId() . ' - ' : '') . $this->getName();
```

I do not know if this change is relevant, ping @orthagh @cedric-anne